src/librbd/api/Snapshot.h

```diff
@@ -27,6 +27,10 @@ struct Snapshot {
 
   static int remove(ImageCtxT *ictx, uint64_t snap_id);
 
+  static int snap_get_name(ImageCtxT *ictx, uint64_t snap_id, std::string *snap_name);
```

Comment:
Nit: stop the `snap_` prefix to these methods (we are in a `librbd::api::Snapshot` namespace so we don't need to call methods `librbd::api::Snapshot::snap_xyz`).